buildAll.proj

```diff
@@ -32,6 +42,17 @@
                       src\Microsoft.ServiceFabric.Services\packages.config;
                       src\Microsoft.ServiceFabric.Services.Remoting\packages.config;
                       src\Microsoft.ServiceFabric.Services.Wcf\packages.config"/>
+ 
+    <TestPackagesConfigs Include="nuprojs\packages.config;
```

Comment:
Can these config list be replaced with $(PackagesConfigs ); instead of copying it here again.
Something like:
<TestPackagesConfigs Include="$(PackagesConfigs); 
src\test\unittests\Microsoft.ServiceFabric.Actors.Tests\packages.config;
src\test\unittests\Microsoft.ServiceFabric.Services.Remoting.Tests\packages.config;
src\test\unittests\Microsoft.ServiceFabric.Services.Tests\packages.config"/>